include/common/debug.h

```diff
@@ -31,12 +33,14 @@ enum {
 #if DEBUG
 	LOG_LEVEL_DEBUG,
 #endif
+	LOG_LEVEL_TEST,
 	LOG_LEVELS
 };
 
 void hexdump(uintptr_t addr, uint32_t bytes);
 void log(const char *fmt, ...) __printf(1, 2);
 noreturn void panic(const char *fmt, ...) __printf(1, 2);
+void run_tests(void);
```

Comment:
This prototype also definitely doesn't go here either

Reply:
done